addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -214,6 +218,14 @@ private void checkMetaPolicy(
                             }
                         };
                 Policy parsedMetaPolicy = parsePolicy(metaPolicy, metaConsumer, msg, id);
+                if (parsedMetaPolicy == null) {
+                    continue;
+                }
```

Comment:
There should be a test for this change.

Reply:
This was also a sonarlint thing, I dunno how theoretical it was. I can try to make it happen and add a test.

Reply:
It's enough by adding one UTF-8 character or one comma.

Reply:
As discussed via IRC this should also be listed as a fix in the CHANGELOG even though it's a rare edge case.